typed-racket-lib/typed-racket/typecheck/signatures.rkt

```diff
@@ -15,6 +15,10 @@
    [cond-contracted tc-expr/t (syntax? . -> . Type?)]
    [cond-contracted tc-expr/t* (syntax? . -> . (values Type? boolean?))]
    [cond-contracted single-value ((syntax?) ((or/c tc-results/c #f)) . ->* . full-tc-results/c)]
+   [cond-contracted tc-sequent-exprs (->* ((listof syntax?) (-> syntax? tc-results/c full-tc-results/c))
```

Comment:
I think "sequent" is not a good term here; it means something else in logic.

Reply:
I agree. Do you have some better names in mind? Actually, I was scrambling with this name. (At first, I called it `tc-exprs-in-sequence-with-propositions-from-preceding-exprs`)

Reply:
`tc-expr-sequence`?